### PR TITLE
chore: Bump version to 6.3.0 w/ changelog

### PR DESCRIPTION
**Pull request by lukasholzer**
### <strong>Pull Request</strong>

<hr>
Hi, thank you for contributing to Barista with this pull request (PR).

To ensure a fast process and merging of your PR please make sure it fulfills the
coding standards and contribution guidelines.

- A feature proposal has been provided, discussed and approved first.
- There is a meaningful description of the issue in GitHub (Screenshots are
  often helpful).
- If the PR introduces breaking-changes or deprecations it matches the following
  guidelines.
  - The commit message follows our commit guidelines.
  - Tests for the changes have been added (for bug fixes / features).
  - Docs have been added / updated (for bug fixes / features).

Please choose the type appropriate for the changes below: <br>

#### Checklist

- [ ] I have read the CONTRIBUTING doc and I follow the PR guidelines
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+  ([c7aca9d](https://github.com/dynatrace-oss/barista/commit/c7aca9d2b366cb1a45162d3590d04b485e747abb))
+- **quick-filter:** Add a new quick filter component inside an experimental
+  package. The quick filter component is used to provide a quick way to operate
+  with the filter field inside a sidebar. Inside the quick filter only an
+  autocomplete with simple options can be displayed.
+  ([dc950e4](https://github.com/dynatrace-oss/barista/commit/dc950e4035c028ab60ca2036d9bc0a56db475b12)),
+  closes [#453](https://github.com/dynatrace-oss/barista/issues/453)
+  [#254](https://github.com/dynatrace-oss/barista/issues/254)
+- **slider:** Adding a new Slider component for number inputs. The Slider
+  essentially works as the html range element.
```
Comment (tomheller) [inline CHANGELOG.md:25]:
```suggestion
- **slider:** Adding a new Slider component.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+  closes [#178](https://github.com/dynatrace-oss/barista/issues/178)
+- **filter-field:** Fixes an issue where two filter field flaps were open at the
+  same time.
+  ([85feb21](https://github.com/dynatrace-oss/barista/commit/85feb21ee7181a3e5b384eba501894aff31744bd)),
+  closes [#840](https://github.com/dynatrace-oss/barista/issues/840)
+
+### Features
+
+- **chart:** Added support for observable as input for options.
+  ([c7aca9d](https://github.com/dynatrace-oss/barista/commit/c7aca9d2b366cb1a45162d3590d04b485e747abb))
```
Comment (tomheller) [inline CHANGELOG.md:16]:
```suggestion
- **chart:** Added support for the `options` input to be an observable.
  ([c7aca9d](https://github.com/dynatrace-oss/barista/commit/c7aca9d2b366cb1a45162d3590d04b485e747abb))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+- **quick-filter:** Add a new quick filter component inside an experimental
+  package. The quick filter component is used to provide a quick way to operate
+  with the filter field inside a sidebar. Inside the quick filter only an
+  autocomplete with simple options can be displayed.
+  ([dc950e4](https://github.com/dynatrace-oss/barista/commit/dc950e4035c028ab60ca2036d9bc0a56db475b12)),
+  closes [#453](https://github.com/dynatrace-oss/barista/issues/453)
```
Comment (ffriedl89) [inline CHANGELOG.md:22]: I think we need to note here that the quickfilter component is not stable!

Comment (lukasholzer) [reply CHANGELOG.md:22]: I noted it under a whole section called experimental package.

**Pull request merged**